index.html

```diff
@@ -176,22 +182,22 @@ <h2 class="p-title">
                 </li>
                 <li>2015</li>
             </ul>
-        </div>
 
-        <p class="intro-about">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-        </p>
+            <p class="about">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+            </p>
 
-        <ul class="languages">
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-        </ul>
-        <div><button class="clickme" type="button">See Project</button></div>
+            <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <div><button class="clickme" type="button">See Project</button></div>
+        </div>
     </section>
-
-    <section class="about-me">
+<section class="my-intro">
+    <div class="about-me">
```

Comment:
- Kindly remove the fixed `width` and `height` for the paragraph that takes a class `connect` because it cause general horizontal scrolling in the desktop version.
![image](https://user-images.githubusercontent.com/24703171/155773687-5288326f-189c-4239-990e-537b86b97b9a.png)
- Kindly split the about section into two blocks the right one is `40%` and the left one `60%`  in the desktop version.
![image](https://user-images.githubusercontent.com/24703171/155773887-6638b9bd-dfd7-48b1-8543-d173e3541bc5.png)